README.md

```diff
@@ -108,7 +106,7 @@ service MessageHub {
 
 ``` protobuf
 message AstarteMessage{
-  Interface interface = 1;
+  string interface_name = 1;
```

Comment:
In Astarte we always use name + major as unique identifier, and I expect multiple programs attached to MessageHub could potentially use different major versions of the same interface. So you should add the `interface_major` field too.

Reply:
I agree with you in the interface install step ( attach node method ), but in case of sending data we use only the `interface_name` . So should we add the ` major identifier` also in the sending method? because the `AstarteMessage` is part of it.

Reply:
Yes it should be added there too, at least `major_version`. The `minor_version` could be used for debugging purposes (e.g. to check if someone tries to publish on a `minor_version` that is not yet installed in AstarteMessageHub itself), but it's not strictly necessary.